core/spec.md

```diff
@@ -1263,6 +1379,72 @@ HTTP body.
 
 ---
 
+### `xid` Resolution API
+
+The resolution API is used to translate an `xid` into a URL that can be used
+to access an xRegistry entity. An implementation of the registry API SHOULD
+provide a resolution API under the path `/resolution`.
+The resolution API MAY also be offered independently of the registry API.
+It can resolve `xids` and federate access to multiple registries.
+In the following, `/` is the root of the resolution API, even if it would be
+provided under `/resolution` as part of an registry.
+
+#### `GET /xid/{xid}` and `HEAD /xid/{xid}`
+
+The placeholder `{xid}` depicts the `xid` to resolve. As `xids` MAY be full
+URIs or URI references, the `xid` MUST be percent-encoded accordingly.
+
+For `GET` and `HEAD` requests, the server MUST respond as follows:
+
+- If the `xid` is not found, the server MUST respond with an HTTP
+  `404 Not Found`.
+- If the `xid` is found and there exists exactly one URL to access the entity,
+  the server MUST respond with an HTTP `308` where the `Location` header
```

Comment:
I'm not sure 308 is correct because what if the resolver service wants to actually redirect to a new location for the resolver service? I wonder if just returning a 200 is good enough.